src/config/fbInit.js

```diff
@@ -9,10 +9,13 @@ import config from "./fbConfig.js";
 
 // Init Firebase
 firebase.initializeApp(config);
-// Init analytics
-if (config.measurementId) {
+/**
+ * Initialize Google Analytics
+ * Once we implement it ourselves, this is not required for now
+ */
+/*if (config.measurementId) {
```

Comment:
I would recommend to comment out this section using //

Reply:
I just removed the entire thing. I won't be using it anymore.